DxDispatch/tools/GenerateParsingHelpers.ps1

```diff
@@ -0,0 +1,330 @@
+param
+(
+    [string]$SchemaFilePath = "$PSScriptRoot\DmlSchema.json",
```

Comment:
Seems DmlSchema.json is not present in the collection of files. So nobody else can actually run this PS1 :b.

Reply:
Argh yes. Let's discuss what we want to do with this file in the future.